src/app/api/deps.py

```diff
@@ -50,7 +50,7 @@ async def get_current_access(security_scopes: SecurityScopes, token: str = Depen
     except (JWTError, ValidationError, KeyError):
         raise unauthorized_exception("Invalid credentials", authenticate_value)
 
-    entry = await crud.get(entry_id=access_id, table=accesses)
+    entry = await crud.get_entry(accesses, access_id)
```

Comment:
Over here, `crud.get` was selected so we can specify the exception "Invalid credentials". If we change to `crud.get_entry`, we're gonna perform a double check. I'm not sure it's good idea

Reply:
Oh I see, I rollback this change